### ISSUE TITLE
New VAMC Facility: Akron West VA Clinic - Awaiting Editor Health Services

### ISSUE THREAD

**Issue by TroyCMSSupport** [department-of-veterans-affairs/va.gov-cms#19131]
## Intake
- [x] What triggered this runbook? (Flag in CMS, Help desk ticket, Product team, VHA Digital Media)
Trigger: **Flag in CMS**

- [x] Link to associated help desk ticket (if applicable)
Help desk ticket: Helpdesk link https://va-gov.atlassian.net/browse/VAHELP-7889

- [x] Name of submitter (if applicable)
Submitter: **Joshua.Wahl2@va.gov**

- [x] If the submitter is an editor, send them links to any relevant KB articles for the VAMC Facility product.
KB articles: [KB Article links](https://prod.cms.va.gov/help/vamc/about-locations-content-for-vamcs/how-do-i-add-a-facility-to-my-health-care-system)

- [x] Link to facility in production:
Facility CMS link: [Facility Link](https://prod.cms.va.gov/northeast-ohio-health-care/locations/akron-west-va-clinic)
Facility API ID: **vha_541QI**

### New VAMC Facility
Please refer to the Knowledge Base article titled "How do I add a facility to my health care system?" for more information: https://prod.cms.va.gov/help/vamc/about-locations-content-for-vamcs/how-do-i-add-a-facility-to-my-health-care-system

#### CMS help desk steps
**Note: If the help desk is waiting on information from the facility staff or editor, add the "Awaiting editor" flag to the facility with a log message that includes a link to this ticket. Remove the flag when the ticket is ready to be worked by the Facilities team. Be sure to preserve the current moderation state of the node when adding or removing the flag.**
- [x] Become aware that the new facility is now on the Facility API (typically, via a Flag, but this may come in as a helpdesk ticket).
- [x] If the editor has followed the steps from the above Knowledge Base article and included which section and VAMC the facility belongs to (i.e. VA Pittsburgh), great!
  - If not, please check with the editor or VHA digital media regarding what section and VAMC it belongs to.
    - [x] Update the Section (default is "VAMC facilities", but it should be a VAMC system in a VISN) and VAMC system field accordingly.
- [x] Communicate with editor (cc VHA Digital Media) to give them go-ahead to complete the content, with this [KB article](https://prod.cms.va.gov/help/vamc/about-locations-content-for-vamcs/how-do-i-add-a-facility-to-my-health-care-system). (See sample notification email below)

<details><summary>Email template </summary>

```

Hello! You should now be able to edit the draft page for this facility, located at [LINK TO NEW FACILITY DRAFT PAGE ON PROD]

Important: Please make sure that all relevant steps listed within the “How do I add a facility to my health care system?” Knowledge Base article have been completed: https://prod.cms.va.gov/help/vamc/about-locations-content-for-vamcs/how-do-i-add-a-facility-to-my-health-care-system

Once finished, please save this page (and all related VAMC Facility Health Service pages) in the moderation state “Draft." Please do not save them as “Published.”

Please let us know when your draft content is complete, so that we can wrap up the technical process from our end before publishing the new facility to VA.gov. Thanks!

```

</details>

- [ ] When editor has prepared content and let help desk know, proceed to the remaining steps.

#### Drupal Admin steps (CMS Engineer or Help desk) _Help desk will complete these steps or escalate to request help from CMS engineering._
- [x] Update the facility **Meta description** field, using the following format: "Get address and hours, parking and transportation information, and health services offered at [facility name]."
- [x] Move the facility link in the health care system menu to its place in the alphabetized list (medical centers first, then clinics).
- [x] Drupal Admin bulk publishes nodes and facility.
- [x] Contact Lighthouse via Slack at #cms-lighthouse channel that this facility requires a canonical link in the following format (replacing the placeholder data with the actual API Id and VA.gov URL):
  - `vha_541QI,https://www.va.gov/northeast-ohio-health-care/locations/akron-west-va-clinic`
- [x] Add the "Awaiting CSV" flag to the facility node with a revision log message that includes a link to this ticket.
- [x] Let Help desk know this has been done, if not done by Help desk.

#### Wait (days or weeks, potentially)
- [x] After the canonical link has been added to the websites.csv and you have confirmation from Lighthouse that the CSV has been deployed, validate that the change has deployed by checking that the Facility Locator has been updated with the new url.
- [ ] Update this ticket with a comment that the CSV change has been deployed.
- [ ] Edit facility node and remove `New facility` and "Awaiting CSV" flags with a revision log message that includes a link to this ticket.

#### CMS Help desk (wrap up)
- [ ] Notify editor and any other stakeholders.

**Comment by TroyCMSSupport** [comment]
@ian-sears @omahane @jilladams  - Awaiting Editor Revisions of DRAFT site

**Comment by TroyCMSSupport** [comment]
@ian-sears @omahane @jilladams - Awaiting Editor Revisions of DRAFT site - contacted Editor on 9/18/24 through Jira ticket

**Comment by ian-sears** [comment]
This is a legitimate "New" additioanl facility, however it is also an "Annex" building attached to the Akron VA Clinic (the main facility). This new facility has separate health services, etc.
10/15/2024: CMS Help Desk reached out again to the Editor to get quicker response (the actual annex facility has been oipen and seeing Veterans) encouraging the Editor to follow the KB article to get this set up.
CMS Help Desk is awaiting staus update and response from teh the Editor (Jashua.Wall) via the CMS HD Jira ticket.

**Comment by TroyCMSSupport** [comment]
@ian-sears @jilladams @omahane -- Awaiting Editor -- needs to add facility health services as of 10/24/2024

**Comment by ian-sears** [comment]
Editor Published this facility prematurely.
Request was made just now to #cms-lighthouse to have this added to the websites.csv
https://dsva.slack.com/archives/C02BTJTDFTN/p1730219512161849?thread_ts=1729289019.897279&cid=C02BTJTDFTN

We are working with the Editor in the Jira ticket [VAHELP-7889](https://va-gov.atlassian.net/browse/VAHELP-7889)

**Comment by ian-sears** [comment]
CSV has been deployed and resolves in the Facility Locator.
Flags have been removed from the node.
However, we left the Awaiting Editor Flag, as **the Editor continues to have not correctly added the Health Services correctly** (understaffed and difficult to reach). We've reached out (again) to the Editor with details and urgency to complete this for Veterans.
**CMS Help Desk will continue to work with the Editor on this unique situation until that is also complete**. However, this is otherwise done, so closing as Resolved.
